SPDX-Licenses.json

```diff
@@ -0,0 +1,278 @@
+        "License": "custom",
+        "LicenseFile": "PhysX/LICENSE.md"
+    },
+    {
+        "PackageName": "poly2tri-7f0487a-rev1-windows",
+        "URL": "https://github.com/jhasse/poly2tri",
+        "License": "BSD-3-Clause",
+        "LicenseFile": "poly2tri/LICENSE"
+    },
+    {
+        "PackageName": "pybind11-2.4.3-rev2-multiplatform",
+        "URL": "https://github.com/pybind/pybind11",
+        "License": "BSD",
+        "LicenseFile": "pybind11/LICENSE"
+    },
+    {
+        "PackageName": "pyside2-qt-5.15.1-rev2-windows",
+        "URL": "https://download.qt.io/official_releases/QtForPython/pyside2",
+        "License": "custom",
+        "LicenseFile": "pyside2/LICENSES.txt"
+    },
+    {
+        "PackageName": "qt-5.15.2-rev4-windows",
+        "URL": "https://code.qt.io/cgit/qt/qt5.git/",
+        "License": "LGPL-3.0",
+        "LicenseFile": "qt/LICENSE"
+    },
+    {
+        "PackageName": "RapidJSON-1.1.0-rev1-multiplatform",
+        "URL": "https://rapidjson.org",
+        "License": "MIT",
+        "LicenseFile": "RapidJSON/license.txt"
+    },
+    {
+        "PackageName": "RapidXML-1.13-rev1-multiplatform",
+        "URL": "http://rapidxml.sourceforge.net/",
+        "License": "MIT",
+        "LicenseFile": "RapidXML/include/rapidxml/license.txt"
+    },
+    {
+        "PackageName": "SPIRVCross-2021.04.29-rev1-windows",
+        "URL": "https://github.com/aws-lumberyard/SPIRV-Cross",
+        "License": "Apache-2.0",
+        "LicenseFile": "SPIRVCross/LICENSE"
+    },
+    {
+        "PackageName": "SQLite-3.32.2-rev3-multiplatform",
+        "URL": "https://www.sqlite.org/",
+        "License": "Public Domain",
+        "LicenseFile": "SQLite/copyright.txt"
+    },
+    {
+        "PackageName": "squish-ccr-deb557d-rev1-windows",
+        "URL": "http://sjbrown.co.uk/2006/01/19/dxt-compression-techniques/",
+        "License": "MIT",
+        "LicenseFile": "squish-ccr/LICENSE.txt"
+    },
+    {
+        "PackageName": "tiff-4.2.0.15-rev3-windows",
+        "URL": "http://www.simplesystems.org/libtiff/misc.html",
+        "License": "libtiff",
+        "LicenseFile": "tiff/COPYRIGHT"
+    },
+    {
+        "PackageName": "v-hacd-2.3-1a49edf-rev1-windows",
+        "URL": "https://github.com/kmammou/v-hacd",
+        "License": "BSD-3-Clause",
+        "LicenseFile": "v-hacd/LICENSE"
+    },
+    {
+        "PackageName": "xxhash-0.7.4-rev1-multiplatform",
+        "URL": "https://github.com/Cyan4973/xxHash",
+        "License": "BSD 2-Clause",
+        "LicenseFile": "xxhash/include/xxhash/LICENSE.txt"
+    },
+    {
+        "PackageName": "zlib-1.2.11-rev5-windows",
+        "URL": "https://zlib.net",
+        "License": "Zlib",
+        "LicenseFile": "zlib/LICENSE"
+    },
+    {
+        "PackageName": "zstd-1.35-multiplatform",
+        "URL": "http://www.zstd.net/",
+        "License": "BSD",
+        "LicenseFile": "zstd/LICENSE"
+    }
+]
```

Comment:
minor, but would it hurt to add a newline at the end for linux?

Reply:
Sure, I can add it here and fix it in the `license_scan.py` script